xml/release-notes.xml

```diff
@@ -311,6 +311,27 @@
 
   </sect2> -->
 
+  <sect2 xml:id="sec.upgrade.kernel">
+   <title>Unification of &sles; and &opensuseleap; kernel packaging</title>
```

Comment:
Unification sounds off when we now have two additional packages in Leap, I'd suggest "alignment":

```suggestion
   <title>Alignment of &sles; and &opensuseleap; kernel packaging</title>
```